src/LocalCoords.cpp

```diff
@@ -10,6 +10,8 @@
  */
 LocalCoords::LocalCoords(double x, double y, double z, bool first) {
   _coords.setCoords(x, y, z);
+  _phi = 0.;
+  _polar = 0.;
```

Comment:
For 2D case, better to set pi/2 default.

Reply:
Good idea, fixed